tasks/utilities/file_handling/task_transfer_pod5_files.wdl

```diff
@@ -0,0 +1,34 @@
+version 1.0
+
+task transfer_pod5_files {
+  input {
+    String pod5_bucket_path             # GCS bucket path containing `.pod5` files (e.g., "gs://your-terra-bucket/pod5_uploads/")
+    Int disk_size = 100
+    Int memory = 32
+    Int cpu = 8
```

Comment:
Recommend decreasing the default cpus and memory for this task since it's not actually copy/transferring files anywhere, but rather just listing them from the input bucket path

Perhaps use 1 cpu as default and 4 GB memory.

Also suggest changing the task name on line 3 as well as the WDL filename, it's a bit misleading. (also means updating the import statement in the workflow WDL file.

last suggestion is to enable premptible VMs as the runtime is likely very short for this task. Even if the VM were pre-empted, it shouldn't take long to run. Your test with a large number of POD5 files only took 2 min to run.